packages/icons/icons.json

```diff
@@ -4373,5 +4373,110 @@
         "tags": "layer, map, reference, point, geography, distance, boundry",
         "group": "miscellaneous",
         "codepoint": 62321
+    },
+    {
+        "displayName": "Block promote",
+        "iconName": "block-promote",
+        "tags": "upgrade, line, chevrons",
+        "group": "interface",
+        "codepoint": 62322
+    },
+    {
+        "displayName": "Cubes",
+        "iconName": "cubes",
+        "tags": "blocks, shapes, hexagons, stack, geometry",
+        "group": "miscellaneous",
+        "codepoint": 62323
+    },
+    {
+        "displayName": "Hexagon",
+        "iconName": "hexagon",
+        "tags": "shape, geometry, polygon",
```

Comment:
Should we consider adding "entity" for internal referencing use cases?

Reply:
Doh, I got so focused on categorization that the original references totally slipped my mind. Added them all in df25fba491e333b467d23ce38349796a0da9dee0